packages/wrangler/src/init.ts

```diff
@@ -620,3 +706,78 @@ async function findPath(
 		});
 	}
 }
+
+async function createWorkerConfig(
```

Comment:
this doesn't mutate, so maybe it should be called `getWorkerConfig`?
```suggestion
async function getWorkerConfig(
```